styles.css

```diff
@@ -211,11 +250,14 @@ li {
 
 .myself {
   background: #fff;
-  position: relative;
+  display: flex;
+  flex-direction: column;
   top: 48px;
   border-radius: 0 10% 0 0;
+  padding-top: 114px;
   padding-left: 24px;
   padding-right: 24px;
+  padding-bottom: 113.85px;
 }
```

Comment:
- [ ] Good job updating your file to get a desktop version of your website, but I think it could be better you readjust again the position in your about myself section to stick to the Figma design as much as possible, Kindly check in the red on the figure below and update it 👍 

![Capture 2](https://user-images.githubusercontent.com/80458199/220891475-64d5dc2e-cf37-4901-85ab-307be5794a85.JPG)